bar-trender-front-end/src/components/ApiLoginForm.js

```diff
@@ -1,234 +1,155 @@
-import React from 'react';
-
-  
+import React from "react";
+import { Redirect } from "react-router-dom";
 
 class POSTLoginForm extends React.Component {
-
-    constructor() {
-
+  constructor() {
     super();
 
     this.state = {
-     login: {email:"client1@email.com",
-      password:"holamundo."},
-
       input: {},
 
-      errors: {}
-
+      errors: {},
     };
 
-     
-
     this.handleChange = this.handleChange.bind(this);
 
     this.handleSubmit = this.handleSubmit.bind(this);
 
     this.handleLogin = this.handleLogin.bind(this);
-
   }
 
-  handleLogin() {
-    alert('A form was submitted: ' + JSON.stringify(this.state));
-
-    // this.props.searchEngine(this.state.term);
-    /* fetch('https://127.0.0.1:8000/authentication/login/', {
-         method: 'POST',
-        // We convert the React state to JSON and send it as the POST body
-        body: JSON.stringify(this.state.login)
-       }).then(function(response) {
-         console.log(response)
-         console.log("SE HA LOGUEADO DE LOCOS")
-         return response.json();
-       });*/
-
-}
+  async handleLogin() {
+    let errors = {};
+    var url = "https://develop-backend-sprint-01.herokuapp.com/v1/authentication/login";
+    // Call to the api with the credentials given by the user
+    const response = await fetch(url, {
+      method: "POST",
+      headers: { apiKey: "apikeytest" },
+      body: JSON.stringify(this.state.input),
+    });
+    if (response.ok) {
+      var r = await response.json();
+      var token = r.token;
+      sessionStorage.setItem("token", token);
+      window.location.href = "/index";
+    } else {
+      const data = await response.blob();
+      this.setState({ loading: false });
+      errors["email"] = "Email o contraseña incorrecta.";
+    }
+    this.setState({
+      errors: errors,
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7138507)
```suggestion
      errors,
```